shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -681,7 +685,10 @@ fun Map<String, Any>.addSysAttrs(
 fun ExpandedAttributesInstances.checkValidity(): Either<APIException, Unit> =
     this.values.all { expandedInstances ->
         expandedInstances.all { expandedAttributePayloadEntry ->
-            getPropertyValueFromMapAsDateTime(expandedAttributePayloadEntry, NGSILD_OBSERVED_AT_PROPERTY) != null
+            getPropertyValueFromMapAsDateTime(expandedAttributePayloadEntry, NGSILD_OBSERVED_AT_PROPERTY) != null &&
+                expandedAttributePayloadEntry.keys.any {
+                    it == NGSILD_PROPERTY_VALUE || it == NGSILD_RELATIONSHIP_HAS_OBJECT
```

Comment:
could not this be avoidable if the attribute instance is transformed to an `NgsiLdAttributeInstance`?